tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/klass/ClassTest.java

```diff
@@ -64,6 +64,11 @@ public void testTypeRef() {
         BRunUtil.invoke(compileResult, "testTypeRefInClass");
     }
 
+    @Test
+    public void testClassWithDefaultsWithoutInitFunc() {
+        BRunUtil.invoke(compileResult, "testClassWithDefaultsWithoutInitFunc");
```

Comment:
Is this name correct? Shouldn't it reflect the reference to the module-level variable?

Reply:
Changed to `testModuleVariableReferencingClass`